docs/en/latest/tutorials/monitor-api-health-check.md

```diff
@@ -109,7 +121,7 @@ To generate some metrics, you try to send few requests to the route we created i
 curl -i -X GET "http://localhost:9080/"
 ```
 
-If you run the above requests a couple of times, you can see from responses that APISX routes some requests to `node2` and while others to `node2`. That’s how Gateway load balancing works!
+If you run the above requests a couple of times, you can see from responses that APISX routes some requests to `node2` and others to `node2`. That’s how Gateway load balancing works!
```

Comment:
Still two `node2` 😅

Reply:
Ah, I have just noticed it. Grammarly does not see this of course.